src/nanoarrow/common/array_test.cc

```diff
@@ -3343,8 +3387,49 @@ TEST(ArrayViewTest, ArrayViewTestGetString) {
   TestGetFromBinary<FixedSizeBinaryBuilder>(fixed_size_builder);
 }
 
-template <typename BuilderClass>
-void TestGetFromBinaryView(BuilderClass& builder) {
+template <typename BuilderClass, typename ValueT>
+void TestGetFromInlinedBinaryView(
```

Comment:
There is some inconsistency between the signature of this test and `TestAppendToInlinedDataViewArray` preceding it. Whereas here we are accepting `GetValueFunc` and `GetValueDataFunc` as arguments, the aforementioned test still takes a template argument.

Should I just move them both to using a function argument instead of a template argument?